Automattic/wp-calypso
# A4A: Replacing --studio-* CSS styles with --color-* within a8c-for-agencies stylesheets

Related to https://github.com/Automattic/automattic-for-agencies-dev/issues/250

## Proposed Changes

* This PR replaces any usage of `--studio-*` color schemes within a8c-for-agencies stylesheets, with `--color-*` versions.
* It also cleans up some favorites styles now that we're not trying to override the agency dashboard component styles.
* Some of the `--color-jetpack` changes may not ultimately be needed - the examples in code are largely for hidden elements (bulk select for example), so on re-adding those we may then notice odd styles and will be able to change them as needed. We may also want to remove some of the jetpack color mapping examples, but I added all for now.
* The previous color of `--studio-automattic-60` didn't exist. I've replaced it with `--color-accent-60`.
* Similarly, `--studio-automattic-80` didn't exist and was used in the table headers in the sites dashboard. I've replaced it with `--color-accent-80` which behaves the same as just inheriting the style.
* Other new color variables include `--color-border-light: var(--studio-white)`, `--color-light-backdrop: var(--studio-white)`, `--color-text-white: var(--studio-white)`, as well as orange and yellow variables.
* For black, Pressable hosting filter buttons now use `--color-surface-backdrop` instead of `--studio-black` (as with their button borders). Elsewhere the only other usages for `--studio-black` are for text color, so I've created `--color-text-black` for that (eg. site preview header title).


## Testing Instructions

For the below testing steps, make sure to run `yarn start-a8c-for-agencies` locally and then visit `http://agencies.localhost:3000/`.

* A lot of CSS has changed, but much of this can be tested by checking the following and comparing with trunk:
* Borderless buttons, for example the actions button next to a daily backup on the backup preview pane.
* Foldable card: on the overview page - the color of the Gridicon down / up chevrpo to expand or shrink the foldable card, plus the foldable card text color.
* A4A Sidebar profile dropdown menu: border and box-shadow.
* Marketplace volume pricing slider: the black 'dot' style.
* Billing details header.
* License details action button background
* Preview pane navigation box-shadow
* Site preview header summary link (the URL)
* Site set favorite icon
* Dataviews filter menu border-bottom
* Sites overview column action button border and text color (eg 'Add' buttons)
* Sites overview stats trend down, up and same
* Sites overview row text
* Sites overview Boost score value (yellow for ok, green for good, red for bad)
* The 'Advanced plan' box at /marketplace/hosting/pressable.
* Sites dashboard, WPcom sites favicon color.
* The preview pane navigation tab background color for the selected tab, on hover (on mobile - which can be replicated by selecting the selected menu item). 
* Some example of the white background changes include the site preview content background color, 
to define the white where used and replace `--studio-white`, the site set favorite icon empty state and hover (not when set).
* For black: Pressable hosting filter buttons, text color in several places such as the site preview header titles.


## Pre-merge Checklist
